src/pages/routes-and-stops/routes-and-stops.component.ts

```diff
@@ -72,6 +74,19 @@ export class RoutesAndStopsComponent {
       }
     }
   }
+  
+  showToast(message: string): void {
+    if (this.toast) {
+      this.toast.dismissAll();
+    }
+  
+  this.toast = this.toastCtrl.create({
```

Comment:
[This vim plugin for TS](https://github.com/leafgarland/typescript-vim) might help?  (I use Atom, which has tons of TS plugins, so I haven't tried any vim ones)